certbot/client.py

```diff
@@ -338,7 +338,14 @@ def obtain_and_enroll_certificate(self, domains, certname):
                 "Non-standard path(s), might not work with crontab installed "
                 "by your operating system package manager")
 
-        new_name = certname if certname else domains[0]
```

Comment:
this is fine though maybe keeping the work in a util function like

`new_name = certname if certname else util.strip_wildcards(domains[0])`

might be cleaner?

Reply:
My personal feeling is stuff in `util` should generally be useful to multiple pieces of the project. Do we currently have another use case in mind for a function like this? Unless we have one now, my vote would be to leave it out of `util` until we do.

I'm happy to add a separate function to do this in `client.py` if that seems cleaner though.

Reply:
Seems slightly cleaner but it's your call about whether it's worth the time :)